qiskit/providers/ibmq/api/clients/experiment.py

```diff
@@ -64,8 +64,9 @@ def experiments(
         Returns:
             A list of experiments.
         """
-        return self.base_api.experiments(
+        resp = self.base_api.experiments(
             backend_name, experiment_type, start_time, device_components, tags)
+        return resp['experiments']
```

Comment:
This (and below) is failing mypy but I'm not sure what I need to do here:

https://travis-ci.com/github/Qiskit/qiskit-ibmq-provider/jobs/403781076

```
qiskit/providers/ibmq/api/clients/experiment.py:69: error: No overload variant of "__getitem__" of "list" matches argument type "str"  [call-overload]

qiskit/providers/ibmq/api/clients/experiment.py:69: note: Possible overload variants:

qiskit/providers/ibmq/api/clients/experiment.py:69: note:     def __getitem__(self, int) -> Any

qiskit/providers/ibmq/api/clients/experiment.py:69: note:     def __getitem__(self, slice) -> List[Any]
```

Reply:
I think I get it. The `base_api.experiments` function says it's returning a List but it's actually returning a Dict now so I need to change that for mypy to work.